app/helpers/services_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ServicesHelper
+  def render_markdown(field)
+    return '' if !field
+
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true)
```

Comment:
We call this 5 times in a row, it might be better to not instantiate a new renderer each time?